### PR TITLE
Orion Fatigue, the latest bing in the military fashion system

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

image

image

Here comes the newest clothing that marines want! Originated from the Orion Military Outpost to serve their mission, this dashing rambo-esqe uniform makes any marines spend their entire paycheck for!

Ooooo, with armor

image

image

## Why It's Good For The Game

Good art demands to be in game. Kuro approves. Thanks to Ghost (IGN:Jack(ie)) for the sprites!

image

## Changelog
:cl: novaepee and Ghost (IGN:Jack(ie))
add: Orion Fatigue! Find it at the nearest clothing vendor!
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
ASS STRAPS

**Comment by MereWalker** [comment]
Too bad we can rappel whatsoever. If we could the straps would be cool. Nonetheless look forward to the drip.

**Comment by Strayed-Raven** [comment]
I will take a future addition of the Sneaking suit from MGS3. Otherwise cannot wait to see how this blends

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -1221,6 +1221,7 @@
 			/obj/item/clothing/under/marine/camo =-1,
 			/obj/item/clothing/under/marine/camo/desert =-1,
 			/obj/item/clothing/under/marine/camo/snow =-1,
+			/obj/item/clothing/under/marine/orion_fatigue =-1,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:1224]:
```suggestion
			/obj/item/clothing/under/marine/orion_fatigue = -1,
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by Alexeivichi** [comment]
METAL GEAR.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**